implementations/rust/ockam/ockam_command/src/enroll/enrollment_token_generate.rs

```diff
@@ -21,17 +21,17 @@ pub struct GenerateEnrollmentTokenCommand {
     #[clap(display_order = 1002, long, default_value = "default")]
     identity: String,
 
-    /// Comma-separated list of attributes
-    #[clap(last = true, required = true)]
-    attributes: Vec<String>,
+    /// Attributes (use '=' to separate key from value)
+    #[clap(value_delimiter('='), last = true, required = true)]
```

Comment:
I thought this was supported by default? i.e. that you could either write `--attrs foo` or `--attrs=foo`? I feel like we should support both

Reply:
The `attrs` argument is defined as a `Vec<String>` where each `String` is a key-value and each entry of the vector will be an occurrence of the argument (i.e. separated by space). So, an example of usage would be `... -- k1=v1 k2=v2`. 

The the `value_delimiter` defaults to `,` as defined [here](https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.value_delimiter) so previously we had to define the attributes like `... -- k1,v1 k2,v2`, so we have to choose one of them.

The good thing about using `=` is that you can parse the key-value pair easily. See [here](https://github.com/build-trust/ockam/pull/2919/files#diff-f9e15f4f9fab6ce0ebcfea91f33f4a60632ba437984d6bbdfb89afbdc49dfb34R52-R56).